charts/datahub/values.yaml

```diff
@@ -18,13 +19,18 @@ datahub-gms:
     initialDelaySeconds: 120
     periodSeconds: 30
     failureThreshold: 8
+  podAnnotations:
```

Comment:
I had to extend the `livenessDelaySeconds` value for datahub-gms as it takes a while to startup. I tried `200` which worked. This is probably a bit extreme and we could find a middle ground.